pyproject.toml

```diff
@@ -75,7 +76,10 @@ uproot-browser = "uproot_browser.__main__:main"
 minversion = "6.0"
 addopts = ["-ra", "--showlocals", "--strict-markers", "--strict-config"]
 xfail_strict = true
-filterwarnings = ["error"]
+filterwarnings = [
+  "error",
+  "ignore:can't resolve package from __spec__ or __package__, falling back on __name__ and __path__:ImportWarning",  # PyPy NumPy
```

Comment:
@mattip, this is an odd warning from NumPy PyPy that gets produced if you enable all warnings (`filterwarnings=["error"]` for pytest, or `-Werror`)

https://github.com/henryiii/uproot-browser/runs/5586825626?check_suite_focus=true

Reply:
I have seen that but I have no idea what it means.